Source/Core/UICommon/AutoUpdate.cpp

```diff
@@ -215,7 +234,14 @@ void AutoUpdateChecker::CheckForUpdate(std::string_view update_track,
   // TODO: generate the HTML changelog from the JSON information.
   nvi.changelog_html = GenerateChangelog(obj["changelog"].get<picojson::array>());
 
-  OnUpdateAvailable(nvi);
+  if (std::getenv("DOLPHIN_UPDATE_TEST_DONE"))
+  {
+    // We are at end of updater test flow, send a message to server, which will kill us.
+    req.Get(
+        fmt::format("{}/update-test-done/{}", GetUpdateServerUrl(), GetCurrentProcessIdAsString()));
+  }
+  else
+    OnUpdateAvailable(nvi);
```

Comment:
You got curlies on the `if`, we need balance in the force.
```suggestion
  else
  {
    OnUpdateAvailable(nvi);
  }
```